src/pages/Swap/LimitOrderBox/constants/index.ts

```diff
@@ -0,0 +1,8 @@
+import { ChainId } from '@swapr/sdk'
+/**
+ * List of supported chainId
+ * At the moment, only Ethereum and Gnosis are supported as CoW API is the
+ */
+export const supportedChainIdList = [ChainId.MAINNET, ChainId.GNOSIS]
+
+export const invalidChars = ['-', '+', 'e']
```

Comment:
This seems live very slim constants I think maybe we can put some stuff inside like max number of minutes for order expiry it seems to be hardcoded

Reply:
Will see what I can add here.